core/src/main/resources/defaults/blueprints/cdp-sdx.bp

```diff
@@ -4,70 +4,128 @@
   },
   "description": "CDP 1.0 Shared Data Experience template with Atlas and Ranger",
   "blueprint": {
-    "cdhVersion": "7.0.0",
+    "cdhVersion": "6.0.99",
```

Comment:
What's the reason for downgrading the version?

Reply:
will be addressed.